app/models/book.rb

```diff
@@ -1,3 +1,9 @@
 class Book < ApplicationRecord
-  
+  validates_presence_of :title
+  has_many :reviews
+
+
+  def average_rating
+    Book.reviews.average(rating)
+  end
```

Comment:
Keep in mind what you know about Ruby. Since you are on an instance of a Book, and reviews is a method available on an instance of a book (even though you've inherited it and not written it yourself) you can call to reviews without needing anything else in front of it. I would want the guts of this method to look more like `reviews.average(:rating)`.  Also note that you were trying to call reviews on the Book class, but a review belongs to a specific book.